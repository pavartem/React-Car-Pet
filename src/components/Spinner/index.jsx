import React from 'react';

const Spinner = () => {
    return (
        <div className="spinner-border text-primary text-center" role="status">
            <span className="sr-only">Loading...</span>
        </div>
    );
};

export default Spinner;
