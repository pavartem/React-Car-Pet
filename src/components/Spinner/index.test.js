import React from "react";
import {configure, shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Spinner from "./index";

configure({adapter: new Adapter()});

describe('<Spinner/>', () => {
    const wrapper = shallow(<Spinner/>);

    it('Should render component', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('Should render right div', () => {
        expect(wrapper.find(<div className="spinner-border text-primary text-center" role="status">
            <span className="sr-only">Loading...</span>
        </div>)).toBeTruthy();
    });

    it('Should render right text', () => {
        expect(wrapper.text()).toEqual('Loading...');
    });
});
