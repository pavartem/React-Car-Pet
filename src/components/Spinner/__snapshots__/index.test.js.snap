// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> Should render component 1`] = `ShallowWrapper {}`;
